src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -152,11 +171,13 @@ public static String getChatMessageForMistakeCount(ToaMistakeTrackerConfig confi
         // Special case a few mistake chat messages based on config
         if (config.mistakeMessageStacking() != StackingBehavior.NONE && mistakeMessage.equals("?")) {
             return getStackingChatMessage(mistakeMessage, mistakeCount);
-        } else if (mistakeMessage.contains("|")) {
-            return getAlternatingChatMessage(mistakeMessage, mistakeCount);
+        }
+        if (mistakeMessage.contains("|")) {
+            return getAlternatingChatMessage(mistakeMessage, mistake.getDefaultMessage(), mistakeCount);
         }
 
-        return mistakeMessage;
+        // Truncate message length to prevent overly-spammy messages taking up too much screen space
+        return truncateMessage(mistakeMessage);
```

Comment:
I think we should do this a single time in the calling method `addChatMessageForMistake`, potentially right before or after the empty string check.